database/seeders/NewTenantSeeder.php

```diff
@@ -93,6 +92,8 @@ public function run(): void
             // Change Request
             ChangeRequestTypeSeeder::class,
             ChangeRequestStatusSeeder::class,
+
+            ContactSeeder::class,
```

Comment:
This should be added to just the regular database seeder I think. This seeder is run when a new tenant is created in every environment including production.

Reply:
If that is the case we'll need to make some other removals from this file in Aiding App and Advising App, as there are non metadata seeders included in each.